### PR TITLE
I288 mousemove fixes

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/alpheios-core/issues/288

>to simplify the Mouse Move (or over) event, could we have an advanced section where we can configure the delay, the accuracy, Google docs container id?

I have re-defined the following options as UI options - `mouseMoveDelay`, `mouseMoveAccuracy`, `enableMouseMoveLimitedByIdCheck`, `mouseMoveLimitedById`

because 
 - they are UI options by sense
 - it would make tabs more clear

now tabs look like the following:

image

image

### CODE REVIEW

File: packages/components/src/lib/selection/media/html-selector.js [hunk 1]
```diff
@@ -256,8 +257,8 @@ export default class HTMLSelector extends MediaSelector {
     }
     // clean string:
     //   convert punctuation to spaces
-    anchorText = anchorText.replace(new RegExp('[' + textSelector.model.getPunctuation() + ']', 'g'), ' ')
 
+    anchorText = anchorText.replace(new RegExp('[' + textSelector.model.getPunctuation() + ']+', 'g'), ' ')
```
Comment (irina060981) [inline packages/components/src/lib/selection/media/html-selector.js:261]: this is the fix for 
> I did also notice that the two digit words are not always selected on mousemove: sometimes it just ignored it, sometimes it does highlighte the line and generate a wrong pop-up, sometimes it does process the two digit word.
see examples below in Google doc

the problem here that, Google docs converts spaces to special characters, 
and they are converted with this Regexp to several spaces, that could braek words definition

**Comment by irina060981** [comment]
> nice catch on the punctuation regex.
> 
> It's okay to merge but I'm not sure about the options change. I was thinking maybe we would have a new "Advanced" options tab, which we could put things like the mousemove and google docs settings that we don't expect people to change, but want to make available in case we need to tell them to.

ok, then I would merge it and we could continue discussion about options tabs later and change it if you think it should be on Advanced.

From my point of view Advanced tab is something like Other tab, noone could be sure - what should be placed there :) that's why I decided to put these settings to UI tab.

**Comment by balmas** [comment]
>From my point of view Advanced tab is something like Other tab, noone could be sure - what should be placed there :) that's why I decided to put these settings to UI tab.

Ok let's see what @monzug thinks

**Comment by monzug** [comment]
I like the Advanced tab where we can put more stuff too (like Log level). otherwise it should go in UI

**Pull request merged**